cfg/rgl_base.cfg

```diff
@@ -80,7 +79,6 @@ sv_gravity "800"                                 // Sets gravity to default (800
 sv_pure "2"                                      // Sets sv_pure to 2. Don't touch this
 sv_pure_kick_clients "1"                         // kick clients violating or attempting to violate sv_pure
 sv_pure_trace "1"                                // sets the server to print a message whenever a client is verifying a CRC for a file
-sv_pure_consensus "5"                            // minimum file hash checks for sv_pure
```

Comment:
sv pure doesn't hash files. the code for this is commented out. this cvar therefore does nothing